TYPO3.Flow/Documentation/TheDefinitiveGuide/PartIII/AspectOrientedProgramming.rst

```diff
@@ -971,6 +971,34 @@ The following example introduces a new interface ``NewInterface`` to the class
 		}
 	}
 
+Trait introduction
+-------------------
+
+Like the interface introductions, also trait introductions are declared
+by annotation. It even uses the same annotation with a different argument:
+
+``@Flow\Introduce("PointcutExpression", traitName="NewTraitName")``
+
+Again only pointcuts filtering for classes make sense. The traitName must
+be a fully qualified "class" (trait) name without leading backslash.
+
+The following example introduces a trait ``SomeTrait`` to the class ``MyClass``.
```

Comment:
I tried to create a trait and add the annotation to the trait class itself, which didn't work. Could imagine others would try to the same to avoid having two classes to introduce a trait. Maybe we should mention that?